tests/load/pipeline/test_merge_disposition.py

```diff
@@ -116,8 +132,125 @@ def data(slice_: slice = None):
     assert eth_2_counts == eth_3_counts
 
 
+@pytest.mark.essential
 @pytest.mark.parametrize(
-    "destination_config", destinations_configs(default_sql_configs=True), ids=lambda x: x.name
+    "destination_config",
+    destinations_configs(
+        default_sql_configs=True,
+        local_filesystem_configs=True,
+        supports_merge=True,
+    ),
+    ids=lambda x: x.name,
+)
+@pytest.mark.parametrize("merge_strategy", ("delete-insert", "upsert"))
+def test_merge_record_updates(
```

Comment:
Added this test to explicitly test record updates, which we didn't do so far in any of the existing tests (at least not beyond row counts).

Reply:
oh wow, thanks for catching this!!